tests/publiccloud/transfer_repos.pm

```diff
@@ -16,6 +16,7 @@ use strict;
 use utils;
 use publiccloud::ssh_interactive "select_host_console";
 use maintenance_smelt qw(is_embargo_update);
+use version_utils "is_sle_micro";
```

Comment:
```suggestion
```

Reply:
Do you mean removing that `use` call? Will `is_...` work the same ?

Reply:
Please remove it. It's not in use in this file.